modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/servlet/taglib/util/BlogsEntryActionDropdownItemsProvider.java

```diff
@@ -314,6 +316,12 @@ private String _getPermissionsURL(BlogsEntry blogsEntry) {
 	}
 
 	private String _getRedirectURL() {
+		String redirect = ParamUtil.getString(_httpServletRequest, "redirect");
+
+		if (Validator.isNotNull(redirect)) {
```

Comment:
@tototrinh 
ParamUtil will return BLANK as default. So I think you should use !Validator.isBlank(...)